packages/datetime/test/dateRangeInputTests.tsx

```diff
@@ -89,6 +147,22 @@ describe("<DateRangeInput>", () => {
         return getEndInput(root).props().value;
     }
 
+    function assertDateRangesEqual(actual: DateRange, expected: string[]) {
+        const [expectedStart, expectedEnd] = expected;
+        const [actualStart, actualEnd] = actual.map((date: Date) => {
+            if (date == null) {
+                return null;
+            }
+            return [
+                date.getFullYear(),
+                padWithZeroes((date.getMonth() + 1) + "", 2),
```

Comment:
Fixed. This was copied from `dateInputTests.tsx`. Went ahead and added a new `toHyphenatedDateString` function to a new `dateTestUtils.tsx` file in `test/common`.